draft-ietf-rats-architecture.md

```diff
@@ -897,3 +899,210 @@ Thomas Hardjono created older versions of the terminology section in collaborati
 Eric Voit provided the conceptual separation between Attestation Provision Flows and Attestation Evidence Flows.
 Monty Wisemen created the content structure of the first three architecture drafts.
 Carsten Bormann provided many of the motivational building blocks with respect to the Internet Threat Model.
+
+# Appendix A: Time Considerations {#time-considerations}
+
+The table below defines a number of relevant events, with an ID that
+is used in subsequent diagrams.  The times of said events might be
+defined in terms of an absolute clock time such as Coordinated Universal Time, 
+or might be defined relative to some other timestamp or timeticks counter.
+
+| ID | Event                       | Explanation of event
+|----|-----------------------------|-----------------------
+| VG | Value generation            | A value to appear in a claim was created
```

Comment:
I think there is a difference between the time a value was generated/created and the time it was collected into a claim. Example: FW is updated on Tuesday but Attesting environment does't run until Wednesday. If the objective is to identify all the points where time could play a role, VG seems to conflate two events.

Reply:
My opinion: I don't think it should be a goal to identify "all the points where time could play a role", just "all the points where the discussion in this section needs to reference in some specific security check for freshness/recency".

That's why I removed events that were not referenced in any check in any example.   Any request to add another row should, in my view, come with proposed text that includes a specific security check equation that requires it.

Reply:
The examples use RATS Roles context to show relevance of the time consideration. The roles architecture doesn't elevate Attesting Environment and Target Environment to the same status (not arguing to do that either). There is however time considerations that apply between TE and AE interactions. The security consideration (which may be contrived) is that the target environment may exhibit one set of claims at value creation and a different set at value collection / measurement. This could happen if attack code has control of the TE and presents one set of claims at time of collection then reverts back to the other set (and presumably continues execution with the attacker's claims). I can convince myself that if the AE and TE were secure by design, that the scenario shouldn't happen. However, it is a common mistake that measurement of code is taken before it is loaded, but modified after being loaded, but the collected evidence always reports what was though to be loaded not what is actually loaded. If it is important to capture this common mistake. Maybe it belongs in security considerations?